src/components/AdminSettings.vue

```diff
@@ -193,8 +213,18 @@ export default {
 				'--color-background-hover': 'var(--color-background-darker)',
 			},
 			redirectUri: window.location.protocol + '//' + window.location.host + generateUrl('/apps/user_oidc/code'),
+			showDeletionConfirmation: false,
+			providerToDelete: null,
 		}
 	},
+	computed: {
+		deletionConfirmationMessage() {
+			if (this.providerToDelete) {
+				return t('user_oidc', 'Are you sure you want to delete the provider "{providerName}"?', { providerName: this.providerToDelete.identifier })
+			}
+			return ''
```

Comment:
```suggestion
			if (!this.providerToDelete) {
				return ''
			}
			return t('user_oidc', 'Are you sure you want to delete the provider "{providerName}"?', { providerName: this.providerToDelete.identifier })
```
a small refactor to reduce the line length, but it's up to you, LGTM otherwise